src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -692,14 +692,10 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
       val g = GroupTable.syntax("g")
       val p: scalikejdbc.QuerySQLSyntaxProvider[scalikejdbc.SQLSyntaxSupport[PolicyRecord], PolicyRecord] = PolicyTable.syntax("p")
 
-      val sg = GroupTable.syntax("sg")
       val pr = PolicyRoleTable.syntax("pr")
       val rr = ResourceRoleTable.syntax("rr")
       val pa = PolicyActionTable.syntax("pa")
       val ra = ResourceActionTable.syntax("ra")
-      val sp = PolicyTable.syntax("sp")
-      val sr = ResourceTable.syntax("sr")
-      val srt = ResourceTypeTable.syntax("srt")
 
       val listPoliciesQuery =
         samsql"""with recursive ${ancestorGroupsTable.table}(${agColumn.parentGroupId}, ${agColumn.memberGroupId}) as (
```

Comment:
The ticket calls for 

> Also the recursive CTE contains the column member_group_id which is not used, remove it

but I'm not sure how to remove the member_group_id column without changing how the recursion works, so I've left it alone for now. Looking for guidance on what exactly this requirement means.

Reply:
the member_group_id column defined by the ancestorsGroupsTable isn't needed. It will make us no longer duplicate recurse. Here, we can just remove the ${agColumn.memberGroupId} from the top this line, and the ${gm/pg.memberGroupId} on lines 202 & 706.
The CTE is selecting the set of unique group id & member group ids i.e. the parent group id and its child. We don't actually care which child group id pulled in the parent group id - we know we're starting from our base user member. As-is, it's selecting extra data about what the child group id is, doing more work and possibly reprocessing the same group on multiple recursions if it's added from multiple children.

The more general change is looking at the SubGroupMemberTable definition and seeing the other places that don't need the memberchild id, but that's out of scope of the ticket. 

Probably easiest to talk it over in a chat/call than through review comments.
I also think it's fine to leave it for a follow on PR of editting the file here, but I'm not picky.

Reply:
There's another recursive query with the same thing going on in this file. Now line 689 I think. Same, we can remove the memberGroupId. Mind touching that one as well?